app/v1/connectors/DesUri.scala

```diff
@@ -16,4 +16,9 @@
 
 package v1.connectors
 
-case class DesUri[Resp](value: String)
+trait BackendUri[Resp] {
+  val value: String
+}
+
+case class DesUri[Resp](value: String) extends BackendUri[Resp]
```

Comment:
Can these be put in a BackendUri companion object so the IntelliJ file looks prettier?